src/main/ruby/core/file.rb

```diff
@@ -312,26 +312,7 @@ def chown(owner, group)
   # by the link). Often not available. Returns number
   # of files in the argument list.
   def self.lchown(owner, group, *paths)
-    raise NotImplementedError, 'lchown not implemented on this platform' unless Truffle::Support::HAVE_LCHOWN
-
-    if owner
-      owner = Truffle::Type.coerce_to(owner, Integer, :to_int)
-    else
-      owner = -1
-    end
-
-    if group
-      group = Truffle::Type.coerce_to(group, Integer, :to_int)
-    else
-      group = -1
-    end
-
-    paths.each do |path|
-      n = POSIX.lchown Truffle::Type.coerce_to_path(path), owner, group
-      Errno.handle if n == -1
-    end
-
-    paths.size
+    raise NotImplementedError, 'lchown not implemented on this platform'
```

Comment:
I would rather remove the first line, since the rest of the logic should work just fine.

Reply:
It's not tested - it's dead code. But ok I'll look at running the super user tests in Docker.